wandb/board/ui/src/containers/HistoryLoader.js

```diff
@@ -88,28 +88,31 @@ export default function withHistoryLoader(WrappedComponent) {
               });
           }
         }
-        nextProps.client.writeQuery({
-          query: FAKE_HISTORY_QUERY,
-          variables: {histQueryKey: nextProps.histQueryKey},
-          data: {
-            model: {
-              id: 'fake_history_query_' + nextProps.histQueryKey,
-              __typename: 'ModelType',
-              buckets: {
-                __typename: 'BucketConnectionType',
-                edges: selectedInfo.map(o => ({
-                  node: {
-                    id: o.id,
-                    name: o.name,
-                    history: o.history,
-                    __typename: 'BucketType',
-                  },
-                  __typename: 'BucketTypeEdge',
-                })),
+        // if graphql is online
+        if (this.props.graphqlStatus) {
```

Comment:
Why does is matter if graphql is online here?

Reply:
My only concern is browser compatibility, but I can agree Opera is not target group. And I think it's the only place.

Reply:
Checking for navigator.online in chrome will still be true when you are disconnect, so I need to come up with a different solution for this.

Reply:
It would be interesting to see where in apollo that error is getting raised, but the easiest solution for now is to use `componentDidCatch` on the ErrorPage boundary and check to see if the error is a network error and displaying the flash if it is, and the error page if it's an error we're not familiar with.

Reply:
@vanpelt It was something funky that had to do with my idea of wrapping entire app into Error boundary. Also, `componentDidCatch` wouldn't catch it as this error wasn't raised by `wtireQuery`, and the pointer above in a screenshot was misleading.
Sorry for waiting the time on this.